src/main/src/main/java/org/geoserver/catalog/impl/WMSLayerInfoImpl.java

```diff
@@ -255,4 +256,16 @@ public void setSelectedRemoteFormats(List<String> selectedRemoteFormats) {
     public void setSelectedRemoteStyles(List<String> selectedRemoteStyles) {
         this.selectedRemoteStyles = selectedRemoteStyles;
     }
+
+    public void reloadRemoteStyles() {
```

Comment:
Does not really need to be part of the interface, the UI can also do "getAllAvailableRemoteStyles().clear()"
and then "getAllAvailableRemoveStyles().addAll(getRemoveStyleInfo())" (using live collections).

I'd also call it just "remoteStyles" without further qualifications.

Reply:
done